LayoutTests/imported/w3c/web-platform-tests/css/selectors/user-valid.html

```diff
@@ -1,14 +1,59 @@
 <!doctype html>
 <title>Support for the :user-valid pseudo-class</title>
+<link rel="author" title="Tim Nguyen" href="https://github.com/nt1m">
 <link rel="help" href="https://drafts.csswg.org/selectors/#user-pseudos">
 <script src="/resources/testharness.js"></script>
 <script src="/resources/testharnessreport.js"></script>
+<script src="/resources/testdriver.js"></script>
+<script src="/resources/testdriver-vendor.js"></script>
+
+<style>
+input {
+  border: 2px solid black;
+}
+
+input:user-valid {
+  border-color: green;
+}
+
+input:user-invalid {
+  border-color: red;
+}
+</style>
+
 <input>
+<input id="user-valid-test" type="email">
```

Comment:
Can we change the IDs for both inputs? (`initially-valid` & `initially-invalid`) The first one initially matches `input.validity.valid` whereas the second one doesn't.